test-git-commands/git-flow-check.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    project = mlrun.get_or_create_project(project_name1, "./")
</pre><p>don't use root context, set it to "./test-git-flow"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>